bin/run_robot.rb

```diff
@@ -40,7 +40,6 @@
   druids = [opts[:druid]]
 end
 
-Dor::WorkflowService.configure(Dor::Config.workflow.url)
```

Comment:
assume the require config/boot above is sufficient to initialize the workflow configuration?

Reply:
Exactly!